src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/PackagePropertyPage.xaml

```diff
@@ -91,16 +91,17 @@
                   HelpUrl="https://go.microsoft.com/fwlink/?linkid=2147240"
                   Category="General" />
 
-  <!-- TODO (tomescht): 
-       We should re-think this. This property is the relative path to the icon file _within the package_; setting it does not guarantee
-       that the corresponding file in the project is correctly copied into the package. A better alternative may be to expose a property
-       with the path to the file in the project instead, and update this one appropriately when that one changes.
-  -->
   <StringProperty Name="PackageIcon"
-                  DisplayName="Icon file"
-                  Description="The relative path from the package root to the icon file. In addition to setting this property you need to make sure that the file is included in the package. Image file size is limited to 1 MB. Supported file formats include JPEG and PNG. An image resolution of 128x128 is recommended."
+                  DisplayName="Icon"
```

Comment:
Updated display name to match that of the similar application icon in the Application property page. Now, they both say `Icon` but are in their appropriate property page. I believe 'file' was there because there used to be an Icon URL, but that functionality is deprecated.